poetry/console/commands/install.py

```diff
@@ -85,15 +85,32 @@ def handle(self):
             # If this is a true error it will be picked up later by build anyway.
             return 0
 
-        self.line(
-            "  - Installing <c1>{}</c1> (<c2>{}</c2>)".format(
-                self.poetry.package.pretty_name, self.poetry.package.pretty_version
+        self.line("")
+        if not self._io.supports_ansi() or self.io.is_debug():
+            self.line(
+                "<b>Installing</> the current project: <c1>{}</c1> (<c2>{}</c2>)".format(
```

Comment:
might be a syntax I am not aware of, but shouldnt this be closed with `</b>`? (similar issue below)

Reply:
It's not a syntax error, no, and it's supported by cleo/clikit: an empty closing tag will close any open tag.

Reply:
how come you only use it for `<b>` and not e.g. the following `<c1>`?